server/lobbyconnection.py

```diff
@@ -507,22 +507,28 @@ def check_version(self, message):
         if response.get('result', '') == 'vm':
             self._logger.debug("Using VM: %d: %s", player_id, uid_hash)
             self.sendJSON(dict(command="notice", style="error",
-                               text="You need to link your account to Steam in order to use FAF in a Virtual Machine. "
-                                    "You can contact an admin on the forums."))
+                               text="You need to link your account to Steam in order to use FAF in a virtual machine. "
```

Comment:
I don't always add linebreaks into my popup messages, but when I do, I add `+` to concat the strings ;-)

Reply:
Except in Python where this is optional and thus redundant :-)

Reply:
when you have a string following a string, the `+` is redundant. But if you have a string followed by a variable, the `+` is mandatory.

So let's say you only have constants multiline strings, w/o `+`. You have 20 of them in the same file. Now you add one variable to one of them - now you have to change the 19 others as well, because consistency? Meh.

This is how it has been done throughout the server code base. I suggest that, if we want to clean it up (hint; I don't ;-)), we do this separately and use string-format instead.

I'll merge this for now since the situation seems painful.